src/AI.hs

```diff
@@ -1,4 +1,170 @@
 module AI where
 
+import Model
+
+
 g:: Int -> String
-g _ = "Success!"
\ No newline at end of file
+g _ = "Success!"
+
+data GameTree a = Empty | Node a [GameTree a] deriving (Show)
+
+data ScoreTree a = GameTree a
+
```

Comment:
Rozumiem że minmaxDepth to jest głębokość na którą jest budowane drzewo, ale to minimumValue i maximumCalue?

Reply:
minimumValue = -100
maximumValue = 100
To są skrajne wartości, które były mi potrzebne do wykonania funkcji znajdującej stan o minimalnym oraz maksymalnym koszcie (wykonuję to przy porównaniu elementów listy, która przechowuje stan gry oraz wyliczoną heurystykę).